anon5r/skytools
# Implement copy-to-clipboard feature for Invitation Codes

# Summary
This PR introduces a new feature that allows users to easily copy Invitation Codes to their clipboard directly from the Invitation Codes page. Upon successful or unsuccessful copy action, a toast notification is displayed, informing the user about the result of their action.

# Changes
1. Created a new Vue.js component, `CopyToClipboard`. This component accepts a `copyText` prop, which is the text to be copied to the user's clipboard upon clicking the component. The component also takes optional `successMessage`, `errorMessage`, `displayDuration`, `position`, and `buttonClass` props for customizing the toast notification and the button style.
2. Integrated the `CopyToClipboard` component into the Invitation Codes page, allowing users to copy invitation codes with a single click.

# Screenshots
<Add relevant screen shots here>

# Test Plan
1. Navigate to the Invitation Codes page.
2. Click "Copy this code!" near to an Invitation Code.
3. Confirm that the Invitation Code is copied to your clipboard.
4. Confirm that a toast notification is displayed in bottom right, indicating the result of the copy action.
5. Check different cases by playing around with the `CopyToClipboard` props, such as different `position` values, and confirm that they work as expected.
